app/styles/components/project-long-description.scss

```diff
@@ -1,10 +1,4 @@
-.project-long-description {
```

Comment:
I didn't remove that because I like to keep the component 1-1 with classes, even if it doesn't have styling.